dotcom-rendering/src/web/components/NewsletterPrivacyMessage.tsx

```diff
@@ -0,0 +1,55 @@
+import { css } from '@emotion/react';
+import { neutral, text, textSans } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+
+const termsStyle = css`
+	${textSans.xxsmall()}
+	color: ${text.supporting};
+	a {
+		${textSans.xxsmall({ fontWeight: 'bold' })};
+		color: ${neutral[0]};
+		text-decoration: underline;
+		:hover {
+			color: ${neutral[0]};
+			text-decoration: underline;
+		}
+	}
+	strong {
+		color: ${neutral[0]};
+		font-weight: bold;
+	}
+`;
+
+export const NewsletterPrivacyMessage = () => (
+	<>
```

Comment:
```suggestion
```

I don't think this fragment is needed

Reply:
You're right, i forgot to remove it when I combined the two spans into one